curp/compute.py

```diff
@@ -298,7 +298,7 @@ def init_current(setting, par):
     # Decide Writer.
     t_0 = time.time()
     
-    logger.info("length of gpair_table: {}".format(len(gpair_table)))
+    logger.info("length of gpair_table: {}".format(len(interact_table)))
```

Comment:
@vantelindome 
`gpair_table` and `interact_table` are not equivalent.
`gpair_table`: group pairs definition
`interact_table`: atom-atom interactions definition

We can output both length of gpair_table and interact_table.

Reply:
Thank you for your comment.
The purpose of this info is to show how much the nonbonded atom-atom interaction table is divided, so I have to change the description.

Reply:
@vantelindome 

>The purpose of this info is to show how much the nonbonded atom-atom interaction table

In this case, logging text should be
`length of interact_table:` not `length of gpair_table`.

Reply:
@passive-radio 
I fixed the text in my latest commit. Please check.